src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Files.kt

```diff
@@ -68,11 +68,13 @@ class Files : FileSystem
         return if (file.extension == "url")
```

Comment:
I guess if the file extension really is '.web.url' for all external link files, then this should be changed too for consistency, though it's a pain .extension won't work for that. Since this seems to be pretty urgent, it's not necessary for merging now but maybe need to remember to update it later.

Reply:
I haven't yet found any information about the difference between .url and .web.url. The dropbox links are .web.url, but the only info I can find is about the .url extension

Reply:
Hmm, well in that case i guess the generic case (e.g. if this is ever being used in another instance, without the Dropbox syncing) would be to remove '.web.url' if present or just '.url' if not. Definitely not for this branch though!